packages/react-core/src/components/Wizard/WizardNavItem.tsx

```diff
@@ -26,16 +28,22 @@ export const WizardNavItem: React.FunctionComponent<WizardNavItemProps> = ({
   isDisabled = false,
   step,
   onNavItemClick = () => undefined,
-  navItemComponent = 'a'
+  navItemComponent = Button
 }: WizardNavItemProps) => {
   const NavItemComponent = navItemComponent as any;
 
   return (
     <li className={css(styles.wizardNavItem)}>
       <NavItemComponent
+        variant="link"
         aria-current={isCurrent && !children ? 'page' : false}
         onClick={() => onNavItemClick(step)}
-        className={css(styles.wizardNavLink, isCurrent && 'pf-m-current', isDisabled && 'pf-m-disabled')}
+        className={css(
+          btnStyles.modifiers.inline,
```

Comment:
In general, I think we try to avoid this sort of thing where `<NavItemComponent>` is also a `<Button>`, but we just use the button component styles. Another way could be for us just to add the `<button>` html element support for use with this BEM element.